pkg/api/v1/node/util.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// TODO: merge with pkg/util/node
+// Package node is deprecated and has been merge with pkg/util/node
```

Comment:
I don't think it is required to keep this file, or it break any existing features?